README.md

```diff
@@ -19,6 +19,17 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 [Power Platform CLI Exposed](https://www.youtube.com/playlist?list=PLlrxD0HtieHhEdLHxQOU96ySSZpMCyAxf)
 
 ## Release Notes
+2.0.1:
```

Comment:
- Add AI-generated code using Power Pages Copilot (preview)

Reply:
For this release, note that this capability is just available for vscode desktop and not web extension